test/dlc_tests/conftest.py

```diff
@@ -864,6 +864,28 @@ def skip_inductor_test(request):
             )
 
 
+@pytest.fixture(autouse=True)
+def skip_torchdata_test(request):
```

Comment:
This fixture is mainly for the training tests, right? Is there a reason for having it on inference PR.